clients/admin-ui/src/pages/login.tsx

```diff
@@ -71,7 +82,26 @@ const useLogin = () => {
   };
 
   if (token) {
-    router.push(INDEX_ROUTE);
+    const configWizardFlagIsActive = Flags.some(
```

Comment:
Since we're already doing the routing on login here, this seemed like the most logical place to me to put this logic.

Reply:
Should this redirection be encapsulated in a custom hook?

Reply:
From what I can tell, in NextJS, router.push is how we've been redirecting routes in the code. 👀